script/Ciel/Day 14/14_01A_CIEL14A_1.txt

```diff
@@ -42,7 +42,7 @@ Someone to whom the boundary between life and death was always blurred.
 -- Page 2, Offset 37610.
 -- 末路は間違いなく、ろくでもない人でなしだ。
 // Everybody's favourite ろくでなし
-Someone fated to be nothing but repugnant human filth.
+In the end, I'm nothing but a repugnant psychopath.
```

Comment:
Okay, but the repetition here was fire

>Someone fated to be nothing but...

If you wanna break up the 'someone's somewhat, then do it on the next line, but I don't think it's really necessary.

Reply:
I think I changed this because it just felt strange to say you're fated to be a psychopath. It just feels more intrinsic, like you already are one or you aren't, rather than human filth which is something you can sink to

Reply:
Sure, but then you can phrase it like

>Someone fated to be nothing more than a...

and it doesn't really matter
the way you rewrote it honestly feels way too simplifying, and it breaks the structure.